contrib/Dockerfile.systemd.test

```diff
@@ -0,0 +1,142 @@
+# This dockerfile is used to test containerd within a container
+#
+# usage:
```

Comment:
How much does this dockerfile differ from the existing Dockerfile? Could we use a single Dockerfile (but possibly stages to be conditional based on build-args?)

Reply:
there will be something different here, because we use systemd as the one process of the container. in this pr https://github.com/containerd/containerd/pull/9350 i resolve part problem, but it's not completely resolved

Reply:
Are more changes expected in the Dockerfile?  Because currently there is only a diff in `COPY` about which directory should be used while copying. I think that can be dealt using build-args as mentioned above.